apps/aevm/src/aevm_eeevm_state.erl

```diff
@@ -237,10 +237,11 @@ do_revert(Us0, Us1, State0) ->
     case vm_version(State0) of
         ?AEVM_01_Sophia_01 ->
             try
-		%% Get the string then convert it to output binary format.
-                {String, State1} = aevm_eeevm_memory:get_area(Us0, Us1, State0),
```

Comment:
Reviewing this line before the change:
* This line considers unallocated words in memory as well defined as 0. That is good.
* This expands memory as needed, and accounts that cost on the REVERT instruction. That is good.